roles/ceph-rgw-loadbalancer/templates/haproxy.cfg.j2

```diff
@@ -29,7 +29,11 @@ defaults
     maxconn                 8000
 
 frontend rgw-frontend
+{% if haproxy_frontend_ssl_termination and radosgw_frontend_ssl_certificate %}
```

Comment:
if a user wants a different TLS certificate on haproxy and radosgw node ?

Reply:
Removed this logic and set a new variable for haproxy specific certificate.